nntrainer/models/model_loader.cpp

```diff
@@ -68,6 +99,7 @@ int ModelLoader::loadOptimizerConfigIni(dictionary *ini, NeuralNetwork &model) {
     std::shared_ptr<ml::train::Optimizer> optimizer =
       createOptimizerWrapped(opt_type, properties);
     model.setOptimizer(optimizer);
+    loadLearningRateSchedulerConfigIni(ini, optimizer);
```

Comment:
what if learning rate scheduler exist but optimizer does not exist?

Reply:
Then it will have no effect as the learning rate scheduler is used only by the optimizer.

Reply:
I think it's reasonable. Could we issue a warning for this case?

If someone add sch in the ini file and add optimizer with model api, they might expect sch is added.

Reply:
There is an existing warning: `There is no [Optimizer] section in the given ini file. This model can only be used for inference`. Do you want one more warning?

Reply:
I see. Maybe adding `[Learning Schedule] section will be ignored if any` their would work. Thanks :)